tests/golos.publication_rewards_tests.cpp

```diff
@@ -1214,19 +1214,39 @@ BOOST_FIXTURE_TEST_CASE(golos_delegators_test, reward_calcs_tester) try {
     BOOST_TEST_MESSAGE("--- create_message: " << name{_users[0]}.to_string());
     BOOST_CHECK_EQUAL(success(), create_message({_users[0], "permlink5"}));
     check();
-    BOOST_CHECK_EQUAL(success(), delegate_vest(_users[11], _users[1], amount, interest_rate_1prcnt));
-    BOOST_CHECK_EQUAL(success(), delegate_vest(_users[12], _users[1], amount, interest_rate_1prcnt));
-    BOOST_CHECK_EQUAL(success(), delegate_vest(_users[13], _users[1], amount, interest_rate));
+    BOOST_CHECK_EQUAL(success(), delegate_vest(_users[11], _users[23], amount, cfg::_1percent));
+    BOOST_CHECK_EQUAL(success(), delegate_vest(_users[12], _users[23], amount, cfg::_1percent));
+    BOOST_CHECK_EQUAL(success(), delegate_vest(_users[13], _users[23], amount, interest_rate));
     check();
-    BOOST_TEST_MESSAGE("--- " << name{_users[1]}.to_string() << " voted");
-    BOOST_CHECK_EQUAL(success(), addvote(_users[1], {_users[0], "permlink5"}, 10000));
+    BOOST_TEST_MESSAGE("--- " << name{_users[23]}.to_string() << " voted");
+    BOOST_CHECK_EQUAL(success(), addvote(_users[23], {_users[0], "permlink5"}, cfg::_100percent));
     check();
-    voter_amount = vest.get_balance_raw(_users[1])["vesting"].as<asset>();
+    voter_amount = vest.get_balance_raw(_users[23])["vesting"].as<asset>();
     delegator1_amount = vest.get_balance_raw(_users[11])["vesting"].as<asset>();
     delegator2_amount = vest.get_balance_raw(_users[12])["vesting"].as<asset>();
     delegator3_amount = vest.get_balance_raw(_users[13])["vesting"].as<asset>();
     produce_blocks(golos::seconds_to_blocks(post.window));
-    BOOST_CHECK_GT(vest.get_balance_raw(_users[1])["vesting"].as<asset>(), voter_amount);
+    std::vector<account_name> dlg_vec_db;
+    std::vector<account_name> dlg_vec_st{_users[11], _users[12], _users[13]};
+    for (auto dlg_obj : vest.get_delegators())
+        if (dlg_obj["delegatee"].as<account_name>() == _users[23])
+            dlg_vec_db.emplace_back(dlg_obj["delegator"].as<account_name>());
+    BOOST_CHECK_EQUAL_COLLECTIONS(dlg_vec_st.begin(), dlg_vec_st.end(), dlg_vec_db.begin(), dlg_vec_db.end());
+    /*
+     * dlg_count = 3
+     * dlg_amount = 5
+     * voter_vest_amount = 500
+     * effective_vesting = 500+5+5+5 = 515
+     *
+     * dlg_reward for _users[11] and _users[12]
+     * interest_rate = 100
+     * reward = 5 * 100 / 515 = 0.97
```

Comment:
please add comment, that `0.97` is rounded to `0`, thats why delegator(s) not added to vote object